.azure/pipelines/e2e-tests.yml

```diff
@@ -44,10 +46,10 @@ phases:
     inputs:
       testRunner: vstest
       testResultsFiles: 'artifacts/logs/**/*.trx'
-- phase: Host_macOS
-  queue:
-    name: Hosted macOS
-    parallel: 4
```

Comment:
Looks like we lost the `parallel` parameters, was that intentional?

Reply:
Hmm, may actually be useful here since we're not using the default-build.yml templates. I'll add `maxParallel: 4` to the strategies where I deleted `parallel: 4`.